apps/taxes/src/lib/theme-synchronizer.test.tsx

```diff
@@ -38,7 +38,8 @@ vi.mock("@saleor/macaw-ui", () => {
 });
 
 describe("ThemeSynchronizer", () => {
```

Comment:
impl of this changed with new macaw, you can delete test and copy-paste impl from other app. eventually this must be extracted to shared

Reply:
It's copied from invoices. I removed the test